tests/test_data/new_parameter_json_files/good_test_rotation_scan_parameters_nomove.json

```diff
@@ -44,7 +44,6 @@
         "beam_size_x": 1.0,
         "beam_size_y": 1.0,
         "focal_spot_size_x": 1.0,
-        "focal_spot_size_y": 1.0,
```

Comment:
How come these parameters are removed on only some of the test jsons?

Reply:
uh, I dunno, but they'll all get deleted soon